fedimint-client/src/sm/executor.rs

```diff
@@ -949,11 +948,10 @@ pub struct ActiveStateKeyBytes {
 }
 
 impl Encodable for ActiveStateKeyBytes {
-    fn consensus_encode<W: std::io::Write>(&self, writer: &mut W) -> Result<usize, std::io::Error> {
-        let mut len = 0;
-        len += self.operation_id.consensus_encode(writer)?;
-        len += writer.write(self.state.as_slice())?;
```

Comment:
BUG. Now clippy complains that `write_all` should be used, as it can see that number of bytes is being discarded. Before it though that we're handling partial writes (while we don't).